src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -525,6 +525,7 @@ static void ParseCommandLineFeatureFlags()
 				}
 			}
 
+#pragma warning disable
```

Comment:
Not needed?

Reply:
It's for the `Regex` below ... need to fix it properly indeed

Reply:
Note: It's a net 8 rule, so I just removed the `#pragma warning disable` for now.